src/main.py

```diff
@@ -8,6 +8,12 @@
 from pathlib import Path
 from config import DEBUG, TOKEN
 
+config = {
```

Comment:
Actually, forgot that there is a `config.py`, so might make more sense to include the id in there as a const (uppercased variable), either as `channel_ids.LOBBY` or `LOBBY_ID`.